packages/manager/apps/cloud/client/app/dbaas/logs/detail/home/logs-home.controller.js

```diff
@@ -55,6 +69,34 @@ class LogsHomeCtrl {
     this.service.load();
   }
 
+  getESObjectCount(serviceName) {
+    return this.getIndexIds(serviceName)
+      .then((indexIds) => {
+        this.countESElements += indexIds.length;
+        return this.getAliasIds(serviceName);
```

Comment:
I think getAliasIds has no dependency on getIndexIds. You should consider making these 2 api calls parallel to improve the performance. You can use $q.all for this.

Reply:
You are right, i completely changed the way i count ES objects by using `runLoaders()`.